source/SIL.AppBuilder.Portal.Frontend/src/data/containers/with-current-organization.tsx

```diff
@@ -26,9 +26,12 @@ export function withCurrentOrganization(InnerComponent) {
   });
 
   function mapRecordsToProps(passedProps) {
+    const { currentOrganizationId: id } = passedProps;
+
+    if (!id || id === '') { return {}; }
```

Comment:
this fixes an issue we were seeing in the logs where orbit would report `record not found: organization: ''` -- because there would never be an org without an id